R/descriptives_helper.R

```diff
@@ -148,10 +148,19 @@ variableCols <- sub.inputForDescriptives[which(sub.inputForDescriptives[,"type"]
   allVar<- list(sc = scaleCol, vc = variableCols)
   allNam<- lapply(allVar, FUN=function(ii) {eatTools::existsBackgroundVariables(dat = dat, variable=ii)})
 
+# wenn es eine fake skala ist, gibt es keine Skalenvariable. Die wird dann jetzt hier kuenstlich erzeugt

```

Comment:
Seems reasonable. However, could there be a less hacky workaround regarding the column name 😄 ? (I am asking and not suggesting, because I do not know one myself)

Reply:
Feel free to finde one. This one is at least reasonably foolproof.